reladomo/src/main/java/com/gs/fw/common/mithra/util/DefaultInfinityTimestamp.java

```diff
@@ -31,6 +31,8 @@
 
     private static final MithraTimestamp DEFAULT_SMALL_INFINITY;
 
+    private static final MithraTimestamp IQ_SMALLDATE_INFINITY;

```

Comment:
Please move this to your own code. There is already a usable date defined for IQ in this class.